src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    public double getBase1() {
+        return base1;
+    }
+
+    public void setBase1(double base1) {
+        this.base1 = base1;
+    }
+
+    public double getBase2() {
+        return base2;
+    }
+
+    public void setBase2(double base2) {
+        this.base2 = base2;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {

```

Comment:
All magic numbers in your code should be constants. The number 0.5 used in the calculation of the area is a magic number and should be extracted as a constant field.